target/linux/ramips/image/rt305x.mk

```diff
@@ -389,6 +392,27 @@ define Device/dlink_dir-320-b1
 endef
 TARGET_DEVICES += dlink_dir-320-b1
 
+define Device/dlink_dir-506l
+  SOC := rt5350
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DIR-506L
+  DEVICE_PACKAGES := jboot-tools kmod-usb2 kmod-spi-dev kmod-usb-serial \
+		kmod-usb-serial-option kmod-usb-net \
+		kmod-usb-net-cdc-ether comgt-ncm \
+		kmod-ledtrig-gpio kmod-ledtrig-netdev kmod-ledtrig-timer
```

Comment:
This should only be a basic set of packages that are essential. Is this true for all of them?

Reply:
Indent for hanging lines should be one tab here BTW

Reply:
This is mobile router for 3g/4g, so cdc, ncm must have for it.
Probably I should add qmi here and lede-ncm,qmi?

Reply:
Do you really need kmod-ledtrig-gpio and kmod-spi-dev?
And probably you can also safely remove kmod-ledtrig-netdev and kmod-ledtrig-timer. It's already enabled in the generic config.

Reply:
No, kmod-ledtrig-netdev and kmod-ledtrig-timer does not enabled by default for any ramips.

Reply:
When I said "generic config" it meant `target/linux/generic/config-*`...